content/posts/July-2021.md

```diff
@@ -15,9 +15,25 @@ is either internal (not user facing) or not finished yet.
   did not react to `/packit test`.
   ([packit-service#1132](https://github.com/packit/packit-service/pull/1132))
 
-## Week 28 (June 12th - July 16th)
+## Week 28 (July 12th - July 16th)
 
 - We have a new API endpoint for obtaining projects of a given forge.
   ([packit-service#1159](https://github.com/packit/packit-service/pull/1159))
 - The bug causing that some projects couldn't be loaded in our dashboard was fixed.
   ([packit-service#1161](https://github.com/packit/packit-service/pull/1161))
+
+## Week 29 (July 19th - July 23th)
+
+- Various improvements on our dashboard was done; mainly:
+  - Projects can be filtered by the forge.
+    ([dashboard#124](https://github.com/packit/dashboard/pull/124))
+  - Table with jobs was unified and cleaned.
```

Comment:
```suggestion
  - Table with jobs has been unified and cleaned.
```

Reply:
@jpopelka Here, I am a bit confused in general. Do we use `past simple`, because it's connected to that week, or `present perfect` because we just speak about the achievements?

Reply:
I was thinking that when I see this sometime later past simple sounds like it's no longer 'unified and cleaned', unlike present perfect.
But you're right, when I see it in the context of being connected to that week then it can be past simple.
So leave it as it is :+1: